src/modules/ROOT/pages/changelog/202303/02.json

```
@@ -207,15 +182,15 @@
     },
     {
       "what": [
-        "missing sessionID param in automation session search."
+        "missing sessionID parameter in automation session search. The sessionID value wasn't taken into consideration when filtering an automation session list."
       ],
       "refs": [
         "https://github.com/cortezaproject/corteza/commit/37035984aed08d1c30c90c59a8aeb9d6d2449e31"
       ]
     },
     {
       "what": [
-        "SMTP_FROM header in env vars."
+        "the value of the SMTP_FROM variable in the server .env variables. The variable wasn't taken into consideration is a value was added to it."
```

```suggestion
        "the value of the `SMTP_FROM` variable in the server `.env` variables. The variable wasn't taken into consideration is a value was added to it."
```

I'm still not quite sure what exactly this does; are we prioritizing `.env` variables now? Can you check with the author?